packages/dashboard/src/services/authServerSideProps.js

```diff
@@ -5,6 +5,14 @@ const isProduction = process.env.NODE_ENV === "production";
 export default function authServerSideProps(getServerSideProps) {
   return function authenticate(context) {
     if (isProduction && (!("ory_kratos_session" in context.req.cookies) || !("skynet-jwt" in context.req.cookies))) {
+      if ("skynet-jwt" in context.req.cookies) {
```

Comment:
I understand it's probably not that important, but this in fact prints wrong information in the case where both fields are **not** present in the cookies. 

That would print: 
`Cookie skynet-jwt is present but ory_kratos_session is not - redirecting to login`
`Cookie ory_kratos_session is present but skynet-jwt is not - redirecting to login`

```
const requiredCookiesSet = "ory_kratos_session" in context.req.cookies && "skynet-jwt" in context.req.cookies
if (isProduction && !requiredCookiesSet) {
    if (!"skynet-jwt" in context.req.cookies) {
        console.log(`Cookie "skynet-jwt" is required but not set - redirecting to login`);
    }
    if (!"ory_kratos_session" in context.req.cookies) {
        console.log(`Cookie "ory_kratos_session" is required but not set - redirecting to login`);
    }
    ...
}
```